### PR TITLE
Update `create-keystone-next-app` to `create-keystone-app`

### PR DESCRIPTION
**Pull request by bladey**
Related PR: https://github.com/keystonejs/keystone/pull/5447

Includes warning message for rename:

image

### CODE REVIEW

File: create-keystone-app/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "create-keystone-next-app",
+  "name": "create-keystone-app",
   "version": "1.2.1",
```
Comment (timleslie) [inline create-keystone-app/package.json:3]: We're going to have to manually change the version number here to match what's currently in the `keystone-5` repo. If we don't do that we'll end up publishing into a version range that already exists.

**Review by timleslie — CHANGES_REQUESTED** [review]

File: create-keystone-app/package.json (hunk 1 above)
Comment (bladey) [reply create-keystone-app/package.json]: Updated to `3.5.2`

**Review by timleslie — APPROVED** [review]
Looks good 👍

**Review by timleslie — CHANGES_REQUESTED** [review]
Let's not ship this until we ship a release which announces the changeover, so that people still using `create-keystone-next-app` know that it's no longer the place to go.

**Comment by bladey** [comment]
> Let's not ship this until we ship a release which announces the changeover, so that people still using `create-keystone-next-app` know that it's no longer the place to go.

This is now done, and will be overwritten by this PR.

**Review by timleslie — APPROVED** [review]
Looks great, let's 🚢

**Pull request merged**